### PR TITLE
drop user token class

### PR DESCRIPTION
**Pull request by AlexandreBossard**

### CODE REVIEW

**Review by dmerejkowsky — APPROVED** [review]

**Review by dmerejkowsky — APPROVED** [review]

File: Identity/PermanentIdentity.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public byte[] UserSecret
             }
         }
 
-        public SecretPermanentIdentity(UserToken userToken, string appId = "")
+        private SecretPermanentIdentity(UserToken userToken, string appId = "")
```
Comment (maximerety) [inline Identity/PermanentIdentity.cs]: What does it have to do with dropping user token class? Split in another commit?

File: Identity.Tests/Helpers.cs [hunk 2]
```diff
@@ -13,9 +13,9 @@ public static class Helpers
         public static string TrustchainPublicKey => "dOeLBpHz2IF37UQkS36sXomqEcEAjSyCsXZ7irn9UQA=";
         public static string UserId => "steve@tanker.io";
 
-        public static string GenerateTestToken()
+        public static string GenerateTestPermanentIdentity()
         {
-            var token = new UserToken(TrustchainId, TrustchainPrivateKey, UserId);
+            var token = new SecretPermanentIdentity(TrustchainId, TrustchainPrivateKey, UserId);
```
Comment (maximerety) [inline Identity.Tests/Helpers.cs]: Rename `token`?

File: Identity.Tests/Identity.cs [hunk 3]
```diff
@@ -9,6 +9,45 @@ namespace Tanker.Tests
     [TestFixture]
     public class IdentityTest
     {
+
+        [Test]
+        public void SignatureAndUserSecretAreValid()
+        {
+            string encodedToken = Helpers.GenerateTestPermanentIdentity();
```
Comment (maximerety) [inline Identity.Tests/Identity.cs]: Keeping the `token` wording everywhere is confusing. Rename to `identity`?

File: Identity/Identity.csproj [hunk 4]
```diff
@@ -7,8 +7,8 @@
     <Company>Kontrol S.A.S.</Company>
     <Product>Tanker</Product>
     <Description>Small library for Tanker Identity generation</Description>
-    <AssemblyVersion>1.1.0</AssemblyVersion>
-    <Copyright>Copyright ©  2019</Copyright>
+    <AssemblyVersion>1.2.0</AssemblyVersion>
+    <Copyright>Copyright ©  2020</Copyright>
```
Comment (maximerety) [inline Identity/Identity.csproj]: Keep `2019`, or change to `Copyright © 2019-2020`. The whole point of copyright is to reference the initial year as a priority date!

Also remove the double space 😉

**Review by maximerety — CHANGES_REQUESTED** [review]

**Review by maximerety — APPROVED** [review]

**Pull request merged**